web/supabase/migrations/20221201092605_response_request_view.sql

```diff
@@ -23,5 +45,5 @@ GRANT ALL ON TABLE public.response_and_request TO postgres;
 GRANT ALL ON TABLE public.response_and_request TO anon;
 GRANT ALL ON TABLE public.response_and_request TO service_role;
 
--- THANK YOU REDDIT https://www.reddit.com/r/Supabase/comments/txq9o9/rls_views_and_functions/
-ALTER VIEW response_and_request OWNER TO authenticated;
\ No newline at end of file
+-- -- THANK YOU REDDIT https://www.reddit.com/r/Supabase/comments/txq9o9/rls_views_and_functions/
+-- ALTER VIEW response_and_request OWNER TO authenticated;
```

Comment:
ahh I know why you did this, but we shouldn't commit this. I will make an issue to solve this